nmz/cli/inspectors/proc.go

```diff
@@ -63,9 +67,29 @@ func (cmd procCmd) Run(args []string) int {
 		return 1
 	}
 
+	var pid int
 	if _procFlags.RootPID <= 0 {
-		log.Critical("pid is not set (or set to non-positive value)")
-		return 1
+		if 0 < len(_procFlags.Cmd) {
+			var err error
+
+			args := strings.Split(_procFlags.Cmd, " ")
+			proc, err := os.StartProcess(args[0], args[1:], &os.ProcAttr{})
+			if err != nil {
+				log.Critical("failed to StartProcess: %s", err)
+				return 1
+			}
+			procState, err := proc.Wait()
+			if err != nil {
+				log.Critical("failed to Wait: %s", err)
+				return 1
+			}
+
+			pid = procState.Pid()
```

Comment:
`proc` (`os.Process`) has PID in its structure https://golang.org/pkg/os/#Process